workflows/autometa.sh

```diff
@@ -145,20 +176,162 @@ autometa-taxonomy-majority-vote --lca $lca --output $votes --dbdir $ncbi
 
 # output:
 # Will write recovered superkingdoms to ${outdir}
-# e.g. ${outdir}/${simpleName}.bacteria.fna
-# e.g. ${outdir}/${simpleName}.archaea.fna
-# e.g. ${outdir}/${simpleName}.taxonomy.tsv
+# e.g. ${outdir}/${prefix}.bacteria.fna
+# e.g. ${outdir}/${prefix}.archaea.fna
+# e.g. ${outdir}/${prefix}.taxonomy.tsv
 
 # script:
+set -x
 autometa-taxonomy \
     --votes $votes \
     --output $outdir \
-    --prefix $simpleName \
+    --prefix $prefix \
     --split-rank-and-write superkingdom \
     --assembly $assembly \
-    --ncbi $ncbi
+    --dbdir $ncbi \
+    --dbtype $dbtype
+{ set +x; } 2>/dev/null
 
-# Step 5: Perform k-mer counting on respective kingdoms
+# Step 5: Taxon-assignment using the GTDB database 
+# NOTE: only performed if `taxa_routine` is 'ncbi_gtdb'
+
+# Step 5.1: Extract bacterial ORFs and run GTDB
+# input:
+# $kingdom_fasta --> Generated by step 4.3
+# $orfs --> User Input
+
+# output:
+# contig_ids --> text file containing metagenome contig IDs classified within NCBI bacteria and archaea
+# orf_ids --> text file containing contig ORF IDs classified within NCBI bacteria and archaea
+# kingdom_orfs --> fasta file containing metagenome ORFs classified within NCBI bacteria or archaea
+# gtdb_input_orfs --> metagenome orfs classified within NCBI bacteria *and* archaea
+
+if [[ "$taxa_routine" == "ncbi_gtdb" ]]
+then
+    echo "Running GTDB taxon assignment step."
+    # output
+    gtdb_input_orfs="${outdir}/${prefix}.orfs.faa"
+    
+
+    for kingdom in ${kingdoms[@]};do
+
+        kingdom_fasta="${outdir}/${prefix}.${kingdom}.fna"
+        
+        contig_ids="${outdir}/${prefix}.${kingdom}.contigIDs.txt"
```

Comment:
This writes sequence headers with the `_` suffix. This should be more appropriately named `{...}.orfPrefixes.txt` and the variable name changed to `orf_id_prefixes`

```suggestion
        orf_prefixes="${outdir}/${prefix}.${kingdom}.orfPrefixes.txt"
```

NOTE: `contig_ids` will need to be changed to `orf_prefixes` where appropriate as well